i18n/insertMissingLocaleKeys.js

```diff
@@ -0,0 +1,94 @@
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+// Convert import.meta.url to __dirname equivalent
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+// Define the path to our i18n locales directory
+const localesI18nDirectory = path.join(
+    __dirname,
+    '/locales'
+)
+
+// Import the key-checking function
+const checkLocaleKeys = () => {
+    const localeFiles = fs
+        .readdirSync(localesI18nDirectory)
+        .filter(file => file.endsWith('.json'))
+
+    const enFile = 'en.json'
+    const enFilePath = path.join(localesI18nDirectory, enFile)
+    const enContent = JSON.parse(fs.readFileSync(enFilePath, 'utf-8'))
+
+    // Extract keys and find missing ones
+    let missingKeys = {}
+    localeFiles.forEach(file => {
```

Comment:
you could do this in a separate PR: 
I think this could be written a little bit simpler. 
1) iterate over the en file keys. 
2) forEach the files and traverse to the key. If it's missing, add the key/value. 
3) done. so easy. 😆  (It's not that easy)